pkg/kubectl/rolling_updater_test.go

```diff
@@ -1572,8 +1574,9 @@ func TestRollingUpdater_readyPods(t *testing.T) {
 			Status: api.PodStatus{
 				Conditions: []api.PodCondition{
 					{
-						Type:   api.PodReady,
-						Status: status,
+						Type:               api.PodReady,
+						Status:             status,
+						LastTransitionTime: now,
```

Comment:
Have this time being fixed and pass two different times (so you will need two different test cases below) - one that is after this time and one that is before.